templates/tsconfig.json.ts

```diff
@@ -6,10 +6,16 @@ export = (answers => {
 	const useTypeChecking = answers.tools && answers.tools.indexOf("type checking") > -1;
 	if (!useTypeScript && !useTypeChecking) return;
 
+	// Minimum supported Node.js version for this adapter.
+	// Change this in sync with package.json
+	const minNodeVersion = 12;
```

Comment:
Please create one global variable for minNodeVersion and not one here and one in package.json.ts

Reply:
... or should we make this a question? (Of course only when the user selects "ask me everything")

Reply:
I like it